src/CSI.IBTA.AuthService/Controllers/AuthController.cs

```diff
@@ -0,0 +1,31 @@
+﻿using CSI.IBTA.AuthService.Interfaces;
+using CSI.IBTA.Shared;
+using Microsoft.AspNetCore.Mvc;
+
+namespace CSI.IBTA.AuthService.Controllers
+{
+    [Route("api/v1/[controller]")]
+    [ApiController]
+    public class AuthController : ControllerBase
+    {
+        private readonly IAuthenticationService _authenticationService;
+
+        public AuthController(IAuthenticationService authenticationService)
+        {
+            _authenticationService = authenticationService;
+        }
+
+        [HttpPost(Name = "Login")]
+        public async Task<IActionResult> Login(LoginRequest request)
+        {
+            var response = await _authenticationService.Login(request);
+
+            if (response == null)
```

Comment:
How it could be null? Authentication service throws exceptions.
I would recomment authentication service return AuthenticationResult ( the class what was mentioned for Administrator Portal) with bool IsSuccess and string Description. 
Login may return Ok ( 200 status ) all the time, or 401 for failed authentication - does not really matter, feel free to choose.